### ISSUE TITLE
Homekit does not see my homebridge with camera

### ISSUE THREAD

**Issue by vlaryn** [KhaosT/homebridge-camera-ffmpeg#290]
My homekit will not see my hub or camera when I try and add them.

I am running Ubuntu 
I have installed ffmpeg and homebridge. 
Below is my config.json
homebridge starts and I even see the message that says TEST-Camera is running on port 42049
I have tested my rtsp string in vlc and my stillimage string in browser and I can see the video and still image.
I have tried deleting my accessories and persist directories and tried changing my bridge name and mac address. Not sure where else to look for issues. I am using a HikVision DS-2CD2032-I

Thanks Wayne B. 

{
        "bridge": {
                "name": "Camerabridge",
                "username": "AA:21:3D:E3:CC:11",
                "port": 51826,
                "pin": "999-05-111"
        },

        "platforms": [{
                "platform": "Camera-ffmpeg",
                "cameras" : [
                        {
                            "name" : "TEST-Camera",
                            "videoConfig" : {
                                "source": "-re -i rtsp://admin:12345@192.168.1.251:554/Streaming/Channels/102/",
                                "stillImageSource": "-i http://admin:12345@192.168.1.251/Streaming/Channels/1/picture",
                                "maxstreams": 2,
                                "maxWidth": 1920,
                                "maxHeight": 1080,
                                "maxFPS": 15
                                                        }
                                        }]
}]
}

**Comment by vlaryn** [comment]
Was able to resolve issues.

**Comment by cjaca** [comment]
How did you solved you problem?

**Comment by vlaryn** [comment]
I actually went a different direction. I finally was able to get some video to show up, but I pulled it from my Blue IRIS Server instead of directly from the camera. I believe the issue was the path I was pointing to not working. Maybe because of authentication. I remember when I finally got it working from Blue IRIS I had to disable Local authentication. I was not really impressed with how it worked in homekit and removed it. It did not seem to update very well.
